src/algebra/big_operators.lean

```diff
@@ -1465,4 +1466,21 @@ begin
   exact sum_lt_top_iff
 end
 
+open opposite
+
+/- Moving to the opposite ring commutes with summing. -/
```

Comment:
```suggestion
/-- Moving to the opposite ring commutes with summing. -/
```
Doc strings need to start with `/--` otherwise they'll just be ordinary block comments and won't get picked up by Lean properly.

I see a number of other instances below too.

Reply:
Thanks!  I think I caught them all.